05/app.js

```diff
@@ -0,0 +1,82 @@
+const formEl = document.querySelector('form');
+
+const labelList = document.querySelectorAll('label');
+
+formEl.addEventListener('submit', validateData);
+
+function validateData(e) {
+	e.preventDefault();
+
+	const firstName = e.target.elements.firstName.value;
+	const lastName = e.target.elements.lastName.value;
+	const avoidChars = /[1-9!@#$%^&*]/g;
+	fullName(firstName, lastName, avoidChars);
```

Comment:
a jak jest pusta wartość to jest ok? :P

Reply:
Jak jest wartość pusta to informuje nas o atrybut require w znaczniku input. Zarówno dla firstName oraz lastName. Z tego powodu nie chciałem robić niepotrzebnej redundancji. 
![image](https://user-images.githubusercontent.com/46199849/135765275-986cc550-8031-4cb0-9436-677901a2f224.png)